src/cli/commands/force/lightning/lwc/test/ui/mobile/run.ts

```diff
@@ -101,6 +101,7 @@ export class Run extends BaseCommand {
             );
         }
 
+        CommonUtils.startCliAction(messages.getMessage('runningUtamTest'));
```

Comment:
bringing back what I had deleted in [this PR](https://github.com/forcedotcom/lwc-dev-mobile/pull/154/files#r1117594304).